draft-ietf-ohai-ohttp.md

```diff
@@ -905,11 +905,12 @@ In this section, a deployment where there are three entities is considered:
 * A relay operates the Oblivious Relay Resource
 * A server operates both the Oblivious Gateway Resource and the Target Resource
 
-Connections between the Client, Oblivious Relay Resource, and Oblivious Gateway
-Resource MUST use HTTPS in order to provide unlinkability in the presence of a
-network observer.  The scheme of the Encapsulated Request determines what is
-used between the Oblivious Gateway and Target Resources, though using HTTPS is
-RECOMMENDED; see {{server-responsibilities}}.
+Requests from the Client to Oblivious Relay Resource and from Oblivious Relay
+Resource to Oblivious Gateway Resource MUST use HTTPS in order to provide
+unlinkability in the presence of a network observer.  The scheme of the
```

Comment:
Not asking for a change, but more offering a comment: now that I'm re-reading this text it occurs to me that this is overly restrictive. Unlinkability only requires HTTPS between client and relay, not between relay and gateway. I don't think we want to get into that nuance, but the phrasing here makes it seem like encryption on both links is necessary and so not strictly true.

Reply:
Yes, I had the same thought when constructing the text.  I know we discussed this in the past too.

We could remove the second half of the requirement, but I think that I'm with you in leaving well alone.

Reply:
Yeah, I'll just resolve this comment.